openpype/settings/entities/schemas/projects_schema/schema_project_global.json

```diff
@@ -27,8 +27,39 @@
                     ]
                 },
                 {
-                    "type": "schema",
-                    "name": "schema_imageio_file_rules"
```

Comment:
Why did this have to change? The old schema seemed to be the fine. Right?

Reply:
I am processing the more explicitly communicated attributes. This will be followed with code changes, of course. 
Proble here was that global project schema was also using the template `schema_imageio_file_rules` but we dont need to have global override toggle on global shema.